src/status_im/contexts/profile/backup_recovery_phrase/view.cljs

```diff
@@ -2,123 +2,167 @@
   (:require
     [clojure.string :as string]
     [native-module.core :as native-module]
-    [quo.context]
+    [quo.context :as quo.context]
     [quo.core :as quo]
     [react-native.core :as rn]
-    [reagent.core :as reagent]
+    [status-im.common.events-helper :as events-helper]
     [status-im.contexts.profile.backup-recovery-phrase.style :as style]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]
     [utils.security.core :as security]))
 
+(def ^:private step-labels
```

Comment:
have you refactored this screen?

Reply:
Yes, I did, mostly refactored to make it faster to render and also use hooks instead of ratoms, and eliminate the usage of a flat list for just 4 items